test/index.test.ts

```diff
@@ -1,4 +1,4 @@
-import greeter from './index';
+import greeter from '../src';
```

Comment:
This change seems contrary to our de-facto standard :thinking: In practice it has always been a mix of both patterns., but we've been moving toward co-located tests recently. Now tests are co-located on both products and in the controllers repo.

Reply:
Hrm. We definitely still keep tests in a separate folder in the majority of our repos. I support the colocation of tests in e.g. the extension because of how big the file tree is, but I like the clean separation between tests and source code for small packages. I could be brought around, but I have to sleep on it, lol.

Reply:
Ah, I had thought we were moving to co-located tests, and the separate test directory pattern was around because we hadn't migrated it yet! Hadn't realized that anyone still preferred it, or was still actively migrating to it.